app/bundles/EmailBundle/Swiftmailer/Transport/AmazonApiTransport.php

```diff
@@ -265,6 +263,7 @@ public function send(\Swift_Mime_SimpleMessage $toSendMessage, &$failedRecipient
                     }
                 },
                 'rejected' => function (AwsException $reason, $iteratorId) use ($evt) {
+                    dd($reason);
```

Comment:
Looks like test code, isn't it?

Reply:
That is correct, I think I missed it. I just removed this line